src/orga/guidelines.rst

```diff
@@ -171,3 +171,72 @@ The guidance that we can provide here is two fold:
+#. The CI checks pass successfully in the required platforms. Exceptions:
+   - ``noarch: python`` can fail in platforms other than Linux (e.g. missing dependency). For non-noarch packages, the failing platform should be skipped via ``skip: true  # [<platform selector>]``
+   - CI times out or runs out of storage because it tries to build all Python versions in the same job. As long as one version passes, that's ok, since they will run individually in the resulting feedstock.
+#. The submission fulfills the `pull request template checklist <https://github.com/conda-forge/staged-recipes/blob/main/.github/pull_request_template.md>`__.
+#. The license has been correctly identified and allows redistribution.
+#. The source should not contain vendored code. If it does:
+   - Package the vendored project separately and specify the needed dependency in the ``requirements`` section. Preferred if the vendored code is needed at runtime.
+   - Allow the vendored code, but make sure the license files are included in the ``about.license`` field. Usually ok if it's only a build-time dependency (e.g. headers-only library)
+
+Recommended:
+
+#. Source should be obtained from a URL that provides a stable tarball (same SHA along time).
+   Git or other SVC repositories should only be used as a last resort.
+#. Host requirements contained in the :ref:`conda-forge pinnings <pinned_deps>` should be *name-only*; i.e. they do not specify a separate version.
+#. Runtime requirements are not pinned too strictly without a justified reason.
+   Thanks to repodata patches, we can afford to be optimistic about lower or upper bounds instead of single-version pins: ``>=1.4.2,<1.5`` is better than ``==1.4.2``.
+#. The package should place its files under standard locations (e.g. executables under ``$PREFIX/bin``), unless a justified reason is provided.
+
+Python-specific details
+-----------------------
+
+Required:
+
+#. ``noarch: python`` packages fulfill the :ref:`required criteria <noarch>` to be considered as such.
+
+Recommended:
+
+#. The package does not accidentally include a ``tests`` (also ``test``, ``_tests``, or similar) top-level package.
+   The list of files is usually printed by ``pip install`` after the ``adding license file`` message.
+   If this happens, upstream should modify their ``setuptools.find_packages()`` usage accordingly.
+#. The modules checked by ``test.imports`` are not empty (this can happen with placeholder ``__init__.py`` files in top-level packages).
+#. The versions reported by ``pip list`` and ``conda build`` logs match.
+#. ``pip check`` passes. See :ref:`pip_check` for more details.
+
+Compiled objects
+----------------
+
+Required:
+
+#. The source does not include compiled files.
+   In principle, all compiled objects need to be generated in the CI, from source.
+   Exceptions to this rule (e.g. binary repackaging) need to be approved explicitly.
+
+Recommended:
+
+#. SONAMEs follow naming recommendations given by upstream.
```

Comment:
We should recommend run exports as needed here too.

Reply:
Done!